lib/assets/javascripts/_modules/search.js

```diff
@@ -35,6 +35,7 @@
       if (window.location.pathname === '/search' && window.location.search) {
         query = getQueryFromURL()
         doSearch(query)
+        doAnalytics()
```

Comment:
This 'restore' commit should be fixed up to the previous commit, as there's little point in removing code only to add it back in straight away.

Reply:
This is still unresolved: could it be fixed up to the previous commit? Usually we try to avoid removing code and then re-adding it in, in the same PR. Better to consolidate it into one good commit. I appreciate the `doAnalytics` function is new, so it's not _exactly_ the same, but that can be added in to the previous commit too.

Reply:
Oh, yep, fixed that now.